docs/recipes/low_freq_proc.qmd

```diff
@@ -0,0 +1,183 @@
+---
+title: "Low-Frequency Processing"
+execute:
+  eval: false
```

Comment:
Is there any way we can remove this? It would be nice if the code is forced to run to ensure the example always works (or the docs won't build).

Reply:
Agreed, but how can I exclude parts such as saving the processed data, creating a new spool, and visualizing other than commenting these lines (lines > 133)?

Reply:
Why not just do them and show the output in the recipe? 

You can use `dascore.example.spool_to_directory` to save the spool to a directory. If you don't pass a path it will use an OS-specific temporary directory. `dascore.examples.random_spool` might be the easiest spool to create contiguous patches (although the data are boring).

Reply:
I have a feeling that this might be a bit confusing, similar to the "fetch" concept in earlier tutorials. However, how about this?

```python
for patch in sp_chunked_overlap:
    # rest of the code

    # Save processed patch 
    pa_lf_name = get_patch_name(pa_lfp_edgeless)
    pa_lf_path = os.path.join("/path/to/output/dir", pa_lf_name)
    # pa_lfp_edgeless.io.write(pa_lf_path, "dasdae") # commented to make this doc executable 

    # Add the processed patch to a list (so later we can create a spool)
    lf_patches = lf_patches.append(pa_lfp_edgeless)


# Visualize the results
```{python}
# Create a spool of LF processed results
# sp_lf = dc.spool(output_data_dir) # commented to make this doc executable 
sp_lf = dc.spool(lf_patches)

# Merge the spool and create a single patch. May need to sub-select before merging to prevent exceeding the memory limit.
sp_lf_merged = sp_lf.chunk(time=None)
pa_lf_merged = sp_lf_merged[0]

# Visualize the results. Try different scale values for better visualization.
pa_lf_merged.viz.waterfall(scale=0.1)
```

Reply:
Looks good